cmd/juicefs/app/juicefs.go

```diff
@@ -18,6 +18,7 @@ package app
 
 import (
 	"os"
+	metricsserver "sigs.k8s.io/controller-runtime/pkg/metrics/server"
```

Comment:
Could you please move 'metricsserver "[sigs.k8s.io/controller-runtime/pkg/metrics/server](http://sigs.k8s.io/controller-runtime/pkg/metrics/server)"' near '[sigs.k8s.io/controller-runtime/pkg/client](http://sigs.k8s.io/controller-runtime/pkg/client)'?

Reply:
fixed